src/elements/content-sidebar/activity-feed/app-activity/AppActivity.scss

```diff
@@ -0,0 +1,64 @@
+@import '../../../common/variables';
+
+.be {
+    .bcs-app-activity-container {
+        position: relative;
+        z-index: 1;
```

Comment:
Is a new stacking context needed?

Reply:
Trying to be consistent with Comment styling